Refresh experiment docs (#647)
mozilla/data-docs: src/concepts/choosing_a_dataset.md

- How many crashes occur each day?
- How many users have installed a specific add-on?

> If you want to know whether a causal link occurs between two events, you can learn more about [experimentation data](experiments.md).

Reviewer: ```suggestion
If you want to know whether a causal link occurs between two events, consider running an [experiment](experiments.md).
```
